### PR TITLE
nocache should be noCache

### PR DESCRIPTION
**Pull request by wanghaoran1988**
https://github.com/openshift/origin/issues/7817

### CODE REVIEW

File: test/extended/fixtures/forcepull-setup.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
 		"strategy": {
 		    "type": "Docker",
 		    "dockerStrategy": {
-			"nocache": true
+			    "noCache": true
```
Comment (bparees) [inline test/extended/fixtures/forcepull-setup.json]: you've got an extra indent here.

**Comment by wanghaoran1988** [comment]
@bparees sorry, it's my vim setting problem,updated.

**Comment by bparees** [comment]
I was thinking about this some more and given that the test was working with "nocache" basically having no effect (because the wrong value was being used), I wonder if instead of fixing the name (and thereby enabling noCache behavior), we should instead remove the nocache line entirely.

@gabemontero wrote this test, i'd like him to take a look and see which approach he thinks is correct.  i'd prefer *not* to set noCache=true if possible because it's going to make the test slower, so if it's not strictly required for the test behavior, we should just remove it.

**Comment by gabemontero** [comment]
@bparees @wanghaoran1988 - yeah, that build config is not for the actually testing of forcepull, but for the custom forcepull builder image we had to create as this test was introduced prior to newer features like build from a docker file.  In fact there was talk of updating this test to leverage that new form of build when build from docker file was introduced.

But bottom line, the use of noCache one way or the other has no bearing on the testing of forcepull  behavior.

I would advocate either removing the line, or converting the test to create the builder image directly from the docker file currently located at https://github.com/gabemontero/forcepull-extended-test-builder, where the dockerfile is stored in openshift/origin/test/extended/fixtures, and getting rid of that repo altogether.

**Comment by bparees** [comment]
@wanghaoran1988 for now let's just get rid of the nocache line.

**Comment by wanghaoran1988** [comment]
@bparees done

**Comment by bparees** [comment]
[testonlyextended][extended:core(forcePull)]

**Comment by bparees** [comment]
[merge]

**Pull request merged**